values.go

```diff
@@ -1493,7 +1502,11 @@ func (v UFix64) MeteredType(common.MemoryGauge) Type {
 }
 
 func (v UFix64) ToGoValue() any {
-	return uint64(v)
+	value, err := strconv.ParseFloat(v.String(), 64)
```

Comment:
But here it is OK to use ParseFloat or can we not do that as per the comment in Fix64?